src/helpers/validation/getFormHasError.ts

```diff
@@ -0,0 +1,7 @@
+export const getFormHasError = ({
+  form,
+  fieldName
+}: {
+  form: { touched: Record<string, any>; errors: Record<string, any> };
```

Comment:
This could be an interface, it's a little difficult to read like this. Also, is it possible to avoid `any`?